### PR TITLE
Rm 109

### PR DESCRIPTION
**Pull request by tcarmeli1**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10046

## Description
Change rm109 to the new format

### CODE REVIEW

File: demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Pack, Playbook]
+
+
+class IsReadmeExistsValidator(BaseValidator[ContentTypes]):
+    error_code = "RM109"
+    description = "Validates if there is a readme file for the content item."
+    rationale = "Insure that we have a README file for uploading to xsoar.pan.dev"
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py:23]:
```suggestion
    rationale = "Ensure that the content item contains additional information about use-cases, inputs, and outputs."
```

File: demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    error_message = "There is no README file for content item from type {0} in pack named '{1}'. Please add relevant README."
+    related_field = "readme"
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.ADDED, GitStatuses.MODIFIED, GitStatuses.DELETED]
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py:27]: Let's remove this two lines
```suggestion
```

File: demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Pack, Playbook]
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py:17]:
```suggestion
ContentTypes = Union[Integration, Script, Playbook]
```
We have a separate validation for packs.
Scripts can also have readmes so let's check that as well (same behaviour as the old validation)

File: demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+
+class IsReadmeExistsValidator(BaseValidator[ContentTypes]):
+    error_code = "RM109"
+    description = "Validates if there is a readme file for the content item."
+    rationale = "Insure that we have a README file for uploading to xsoar.pan.dev"
+    error_message = "There is no README file for content item from type {0} in pack named '{1}'. Please add relevant README."
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py:24]:
```suggestion
    error_message = "The {0} doesn't have a README file. Please add a README.md file in the content item's directory."
```
Where 0 is the content item type.

File: demisto_sdk/commands/validate/tests/RM_validators_test.py [hunk 5]
```diff
@@ -255,3 +257,45 @@ def test_IsPackReadmeNotEqualPackDescriptionValidator_valid():
...
+            [
+                create_integration_object(),
+                create_integration_object(),
+            ],
+            1,
+            ["There is no README file for content item from type Integration in pack named 'pack_22'. Please add relevant README."]
+        )
+    ]
+)
+def test_IsReadmeExistsValidator_is_valid(content_items, expected_number_of_failures, expected_msgs):
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/RM_validators_test.py:290]: Make sure to fix the tests and add given-when-then doc strings.

**Review by YuvHayun — DISMISSED** [review]
Looks good!
minor comments.

File: demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    related_field = "readme"
+    related_file_type = [RelatedFileType.README]
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message.format(
+                    content_item.content_type, get_pack_name(content_item.path)
+                ),
```
Comment (BEAdi) [inline demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py:32]: The `error_message` above has only one placeholder, here you supply 2 strings.

**Review by BEAdi — COMMENTED** [review]
Looks great

**Review by jbabazadeh — APPROVED** [review]
Nice!
please fix the unit tests that failed.

**Pull request merged**